superset-frontend/src/dashboard/components/PropertiesModal/index.tsx

```diff
@@ -161,10 +161,15 @@ const PropertiesModal = ({
 
       form.setFieldsValue(dashboardInfo);
       setDashboardInfo(dashboardInfo);
-      setJsonMetadata(metadata ? jsonStringify(metadata) : '');
       setOwners(owners);
       setRoles(roles);
       setColorScheme(metadata.color_scheme);
+
+      // temporary fix to remove positions from dashboards' metadata
+      if (metadata && metadata.positions) {
```

Comment:
```suggestion
      if (metadata?.positions) {
```

Reply:
Actually, since we're using `metadata.color_scheme` in line 166, that means that metadata object exists and we don't need the null check at all. So I think that we can remove the `if` and just run `delete metadata.positions`